src/Host/Broker/Impl/Resources.resx

```diff
@@ -237,4 +237,7 @@
   <data name="Trace_UserProfileDirectory" xml:space="preserve">
     <value>User {0} profile directory: {1}</value>
   </data>
+  <data name="Exception_MaxAllowedUsers" xml:space="preserve">
+    <value>Broker has reached maximum allowed concurrent user limit.</value>
```

Comment:
I see that we throw ArgumentOutOfRange exception. If you try it, does it get propagated to the client via BrokerApiError? If so, then there should be error code that will map to the appropriate message in REPL. See `BrokerApiErrorException`